pyproject.toml

```
@@ -166,7 +167,8 @@ dev = [
     "mkdocstrings[python]>=0.29.1,<0.30",
     "mkdocs-gen-files>=0.5.0,<0.6",
     "material-plausible-plugin>=0.3.0,<0.4",
-    "mdx-truly-sane-lists>=1.3,<2"
+    "mdx-truly-sane-lists>=1.3,<2",
+    "build"
```

Nit. Just so we truly only ever touch one line when adding something. Had to look twice what about the dependency above changed.

```suggestion
    "build",
```